src/components/screens/OperatingCompany.tsx

```diff
@@ -29,11 +31,11 @@ export const OperatingCompany = ({ title, operatingCompany, openWebScreen }: Pro
         <Title accessibilityLabel={`${title} (Überschrift)`}>{title}</Title>
       </TitleContainer>
       {device.platform === 'ios' && <TitleShadow />}
+      {!!logo && <Logo source={{ uri: logo }} />}
```

Comment:
logo rendering should be the same as in existing components, inside the wrapper, `Event` for example:

```
<Wrapper>
  {!!logo && <Logo source={{ uri: logo }} />}

  <InfoCard
    category={category}
    addresses={addresses}
    contacts={contacts}
    webUrls={webUrls}
    openWebScreen={openWebScreen}
  />
</Wrapper>
```

Reply:
done in e7cc835bf306052d8dfa8e6ec5856e9a44507369